docs/source/conf.py

```diff
@@ -165,7 +165,7 @@
 # Useful aliases to avoid repeating long URLs.
 extlinks = {
     'github-examples': (
-        'https://github.com/enthought/pyface/tree/master/examples/%s',
+        'https://github.com/enthought/pyface/tree/main/examples/%s',
```

Comment:
same comment here as in traitsui - i think we should be linking to the specific version/tag instead of just linking to main branch. will fix this in a separate PR.